faster_coco_eval/core/cocoeval.py

```diff
@@ -512,7 +551,7 @@ def _summarizeKps():
         if not self.eval:
             raise Exception("Please run accumulate() first")
         iouType = self.params.iouType
-        if iouType == "segm" or iouType == "bbox":
+        if iouType == "segm" or iouType == "bbox" or iouType == "boundary":
```

Comment:
I think something like **if iouType in ['bbox'...]**

Reply:
Done :)